projects/user-telemetry-client/tsconfig.json

```diff
@@ -1,7 +1,8 @@
 {
   "compilerOptions": {
-    "target": "ES2019",
-    "module": "commonjs",
+    "target": "ES2020",
+    "module": "ES2020",
```

Comment:
JS modules and packages are definitely not my area of expertise, but I think typical best practice is to support ESM and CommonJS. 

In this context, it seems reasonably to export only ESM in inline with what we might expect of our own consuming code. However, it could be a breaking change for a hypothetical consumer and should justify an appropriate release.

IIUC `chore` won't trigger any release. 

Sources:
https://github.com/guardian/recommendations/blob/main/npm-packages.md#compiling

Reply:
👍  [d772a90](https://github.com/guardian/editorial-tools-user-telemetry-service/pull/53/commits/d772a906ac3da317448931da8b158d1e7df5ed6e) adds CJS support.